.travis.yml

```diff
@@ -1 +1,22 @@
 language: crystal
+crystal:
+  - latest
+  - nightly
```

Comment:
Do we maybe want to also set the version explicitly?

Reply:
I wish you could, but I could not figure out a way to do it :(